editor/question-ctrl.js

```diff
@@ -20,6 +20,13 @@ angular.module('org.ekstep.question', ['org.ekstep.metadataform'])
 	$scope.questionData = {'questionMaxScore': 1};
 	$scope.questionData.isShuffleOption = false;
 	$scope.questionData.isPartialScore = true;
+  $scope.templateIcons = [];
+  _.each($scope.templatesType, function(template, key){
+    $scope.templateIcons.push({
+      "icon": ecEditor.resolvePluginResource(instance.manifest.id, instance.manifest.ver, 'assets/'+template+'.png'),
```

Comment:
Dont bind the template name with image name. Define the object on top & use it for vertical, horizontal & grid.